spec/support/fake_analytics_spec.rb

```diff
@@ -0,0 +1,350 @@
+require 'rails_helper'
+
+RSpec.describe FakeAnalytics do
```

Comment:
some specs for FakeAnalytics (pii logging, documenting) behavior are in the `analytics_spec.rb`, WDYT of picking those up and moving them here?

https://github.com/18F/identity-idp/blob/76be4ab0b8c6e11e47719ebffc10fd52b52d9aaf/spec/services/analytics_spec.rb#L113-L164

and

https://github.com/18F/identity-idp/blob/76be4ab0b8c6e11e47719ebffc10fd52b52d9aaf/spec/services/analytics_spec.rb#L185-L256